src/Uno.UI.Runtime.Skia.Gtk/buildTransitive/Uno.UI.Runtime.Skia.Gtk.targets

```diff
@@ -4,7 +4,7 @@
 	<Target Name="_UnoSkiaGtkFeatureDefines" AfterTargets="PrepareForBuild">
 
 		<PropertyGroup>
-			<DefineConstants>$(DefineConstants);UNO_REFERENCE_API;HAS_UNO_SKIA;HAS_UNO_SKIA_GTK</DefineConstants>
```

Comment:
This is a breaking change, we cannot know if developers are using that or not...

Reply:
@jeromelaban Is it specifically about Gtk?